### PR TITLE
Update from master

### PR DESCRIPTION
**Pull request by fore5fire**
Updating CDN SDK for version 2020-04-15

### CODE REVIEW

File: sdk/cdn/Microsoft.Azure.Management.Cdn/src/Generated/CdnManagementClient.cs [hunk 1]
```diff
@@ -59,6 +59,17 @@ public partial class CdnManagementClient : ServiceClient<CdnManagementClient>, I
         /// </summary>
         public string ApiVersion { get; private set; }
 
+        /// <summary>
+        /// Azure Subscription ID.
+        /// </summary>
+        public string SubscriptionId1 { get; set; }
```
Comment (isra-fel) [inline sdk/cdn/Microsoft.Azure.Management.Cdn/src/Generated/CdnManagementClient.cs:65]: There's definitely something wrong here. Please make sure the SDK is generated by running `generate.ps1` and your [build tools are up to date.](https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md#updating-build-tools)

Besides, please point to me the link to your openapi spec review PR. Thanks

**Comment by isra-fel** [comment]
Hi @lsmith130 I just noticed this PR was created from a SDK automation branch. Please never do that, because that branch changes often whenever there's change to openapi spec (as you can see it's force-pushed just now).
Please follow our guideline to generate manually: https://github.com/Azure/adx-documentation-pr/blob/master/engineering/adx_netsdk_process.md#generating-net-sdk
Thanks

**Comment by lsmith130** [comment]
@isra-fel, thanks for the info. It might be a good idea to remove the message from the PR that says to open a PR from the automation and provides a link to do so.

**Comment by isra-fel** [comment]
> @isra-fel, thanks for the info. It might be a good idea to remove the message from the swagger PR that says to open a PR from the automation and provides a link to do so. For example the automated comment in my azure-rest-api-specs PR has a link that says "Release SDK Changes" when you expand it: [Azure/azure-rest-api-specs#9433 (comment)](https://github.com/Azure/azure-rest-api-specs/pull/9433#issuecomment-627490520)

Sounds reasonable to me. @NullMDR hello Phoenix, can we update the `Release SDK Change` label just for .NET SDK? Thanks

**Pull request closed without merging**